Automattic/jetpack
# Connect in Place: make the in place flow the default one so it gets more testing

requested by @gravityrail  :)


**Testing Instructions**

*  Run this branch
* Confirm that you always fall under the "in place" flow
* set `define( 'JETPACK_SHOULD_USE_EXTERNAL_CONNECTION_FLOW', false );` and confirm that you fall under the "normal" flow
